### PR TITLE
Remove the need to install genesis creator.

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

Remove the need for installing the genesis creator. It is bundled in the binary.

## Checklist

- [x] My code follows the style of this project.
- [ ] The code compiles without warnings.
- [ ] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: src-tauri/Cargo.toml [hunk 1]
```diff
@@ -13,21 +13,22 @@ edition = "2021"
...
-tauri = { version = "1.4", features = [ "api-all"] }
+tauri = { version = "1.4", features = ["api-all"] }
 serde = { version = "1.0", features = ["derive"] }
 reqwest = "0.11"
 tokio = { version = "1", features = ["full"] }
 dirs = "3.0"
 serde_json = "1.0"
 nix = { version = "0.27.1", features = ["signal", "process"] }
-concordium-rust-sdk = "3.0.1"
+concordium-rust-sdk = { git = "https://github.com/Concordium/concordium-misc-tools.git", version = "3.0.1" }
```
Comment (EmilLa1) [inline src-tauri/Cargo.toml:23]: this looks wrong

Comment (EmilLa1) [inline src-tauri/Cargo.toml:23]:
```suggestion
concordium-rust-sdk = { git = "https://github.com/Concordium/concordium-rust-sdk.git", version = "3.0.1" }
```
?

Comment (abizjak) [reply src-tauri/Cargo.toml:23]: No, I want the one that is available t hrough the misc-tools since the genesis-creator also depends on it.

Comment (abizjak) [reply src-tauri/Cargo.toml:23]: Otherwise we'd have duplicate dependencies.

Comment (EmilLa1) [reply src-tauri/Cargo.toml:23]: Ah i see

**Review by EmilLa1 — APPROVED** [review]

**Review by gachouchani1999 — APPROVED** [review]
It looks good to me and a better idea on the long run for compatibility and user experience

**Pull request merged**